components/jms-mq-support/org.wso2.carbon.cache.sync.jms.manager/ACTIVEMQ_README.md

```diff
@@ -0,0 +1,55 @@
+### How to Build
+1. Build the maven project and copy the jar file to the `<IS_HOME>/repository/components/dropins` directory.
+2. Add the following identity.xml.j2 template configurations
+```yaml
+  {% if cache_invalidator.mb.enabled is defined %}
+    <CacheInvalidator>
+      <MB>
+        <Enabled>{{cache_invalidator.mb.enabled}}</Enabled>
+        <Type>{{cache_invalidator.mb.broker_type}}</Type>
+        <InitialNamingFactory>{{cache_invalidator.mb.initial_naming_factory}}</InitialNamingFactory>
+        <ProviderURL>{{cache_invalidator.mb.provider_url}}</ProviderURL>
+        <TopicName>{{cache_invalidator.mb.topic_name}}</TopicName>
+        <ProducerName>{{cache_invalidator.mb.producer_name}}</ProducerName>
+        <Username>{{cache_invalidator.mb.username}}</Username>
+        <Password>{{cache_invalidator.mb.password}}</Password>
+        <HybridMode>{{cache_invalidator.mb.hybrid_mode_enabled}}</HybridMode>
+      </MB>
+    </CacheInvalidator>
+  {% endif %}
+```
+**Note:** For **IS 5.11.0** this should have been already added. 
```

Comment:
This is available only in the updated pack right? Shall we mention that with some details to identify the timestamp of the pack

Reply:
The update is yet to be released, lets update this in another PR after the release